Core/src/org/sleuthkit/autopsy/datasourceprocessors/VolatilityProcessor.java

```diff
@@ -247,29 +247,32 @@ private void lookupFiles(Set<String> fileSet, String pluginName) {
                     resolvedFiles = fileManager.findFiles(fileName, filePath); //NON-NLS
                 }
                 resolvedFiles.forEach((resolvedFile) -> {
-                    try {
-                        String MODULE_NAME = "VOLATILITY";
-                        BlackboardArtifact volArtifact = resolvedFile.newArtifact(BlackboardArtifact.ARTIFACT_TYPE.TSK_INTERESTING_FILE_HIT);
-                        BlackboardAttribute att1 = new BlackboardAttribute(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_SET_NAME, MODULE_NAME,
-                                "Volatility Plugin " + pluginName);
-                        BlackboardAttribute att2 = new BlackboardAttribute(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_COMMENT, MODULE_NAME,
-                                "Volatility Plugin " + pluginName);
-                        volArtifact.addAttribute(att1);
-                        volArtifact.addAttribute(att2);
-
+                    // check for slack files and do not include them may double the list of files found.
+                    if (!resolvedFile.getName().endsWith("-slack")){
```

Comment:
If you merge with upstream, then you'll get my version of this fix which is to check the file type.

Reply:
Will do this and fix the above.